doc/get_started_on_minikube.md

```diff
@@ -1,58 +1,62 @@
+1. Start a Kubernetes cluster:
 
    ```bash
-   minikube start --kubernetes-version v1.15.0 --cpus 4 --memory "8192mb" # we recommend that you allocate enough RAM (more than 8192 MiB) to the VM
+   minikube start --kubernetes-version v1.15.0 --cpus 4 --memory "8192mb"
    ```
 
-2. Install Helm
+    > **Note:**
+    >
+    > It is recommended to allocate enough RAM (more than 8192 MiB) to the Virtual Machine (VM) using the `--cpus` and `--memory` flag.
+
+2. Install Helm:
 
    ```bash
    curl https://raw.githubusercontent.com/helm/helm/master/scripts/get | bash
    helm init
    ```
 
-3. Check whether the Helm tiller pod is running.
+3. Check whether the Helm tiller pod is running:
 
    ```bash
    kubectl -n kube-system get pods -l app=helm
    ```
 
 ## Step 2: Install Chaos Mesh
 
-Run the following comments to install Chaos Mesh;
+Run the following comments to install Chaos Mesh:
 
 ```bash
 git clone --depth=1 https://github.com/pingcap/chaos-mesh && \
 cd chaos-mesh
 ./install.sh
 ```
 
->**Note:** 
+>**Note:**
 >
-> `install.sh` is a shell script to automate the installation process. To deploy manually,  refer to [Install Chaos Mesh manually](deploy.md).
+> `install.sh` is a shell script to automate the installation process. See [Install Chaos Mesh manually](deploy.md) for more details.
 
-After executing the above commands, you should be able to see the prompt that Chaos Mesh is installed successfully. Otherwise, please check the current environment according to the prompt message or send us an [issue](https://github.com/pingcap/chaos-mesh/issues) for help. 
+After executing the above commands, you should be able to see the prompt that Chaos Mesh is installed successfully. Otherwise, check the current environment according to the prompt message or send us an [issue](https://github.com/pingcap/chaos-mesh/issues) for help.
 
-> **Limitations:**
->
-> There are some known restrictions for Chaos Operator deployed on `minikube` clusters:
-> - `netem chaos` is only supported for `minikube` clusters >= version 1.6.
->
->   In `minikube`, the default virtual machine driver's image doesn't contain the `sch_netem` kernel module in smaller versions. You can use `none` driver (if your host is Linux with the `sch_netem` kernel module loaded) to try these chaos actions on `minikube` or [build an image with sch_netem by yourself](https://minikube.sigs.k8s.io/docs/contributing/iso/).
+## Limitations
+
+There are some known restrictions for Chaos Operator deployed in the Minikube cluster:
+
+- `netem chaos` is only supported for Minikube clusters >= version 1.6.
+
+In Minikube, the default virtual machine driver's image doesn't contain the `sch_netem` kernel module in earlier versions. You can use `none` driver (if your host is Linux with the `sch_netem` kernel module loaded) to try these chaos actions using Minikube or [build an image with sch_netem by yourself](https://minikube.sigs.k8s.io/docs/contributing/iso/).
```

Comment:
> or [build an image with sch_netem by yourself](https://minikube.sigs.k8s.io/docs/contributing/iso/)

This is a broken link.

Reply:
Thanks for this! The link has been updated to `https://minikube.sigs.k8s.io/docs/contrib/building/iso/`. Could you please update it in this PR?

Reply:
Sure.